guides/v2.1/install-gde/prereq/nginx.md

```diff
@@ -16,8 +16,8 @@ Magento supports nginx 1.8 (or the [latest mainline version](http://nginx.org/en
 
 Installation instructions vary based on which operating system you're using:
```

Comment:
The bullet numbering in this section, under "For this example..." are improperly numbered. Please revise.

Reply:
Above changes is done